docs/standard/base-types/standard-numeric-format-strings.md

```diff
@@ -261,12 +266,15 @@ Standard numeric format strings are used to format common numeric types. A stand
   
 <a name="RFormatString"></a>   
 ## The Round-trip ("R") Format Specifier  
- The round-trip ("R") format specifier is used to ensure that a numeric value that is converted to a string will be parsed back into the same numeric value. This format is supported only for the <xref:System.Single>, <xref:System.Double>, and <xref:System.Numerics.BigInteger> types.  
-  
- When a <xref:System.Numerics.BigInteger> value is formatted using this specifier, its string representation contains all the significant digits in the <xref:System.Numerics.BigInteger> value. When a <xref:System.Single> or <xref:System.Double> value is formatted using this specifier, it is first tested using the general format, with 15 digits of precision for a <xref:System.Double> and 7 digits of precision for a <xref:System.Single>. If the value is successfully parsed back to the same numeric value, it is formatted using the general format specifier. If the value is not successfully parsed back to the same numeric value, it is formatted using 17 digits of precision for a <xref:System.Double> and 9 digits of precision for a <xref:System.Single>.  
-  
- Although you can include a precision specifier, it is ignored. Round trips are given precedence over precision when using this specifier.  
-  
+ The round-trip ("R") format specifier attempts to ensure that a numeric value that is converted to a string is parsed back into the same numeric value. This format is supported only for the <xref:System.Single>, <xref:System.Double>, and <xref:System.Numerics.BigInteger> types.  
+
+For a <xref:System.Double> values, the "R" format specifier in some cases fails to successfully round-trip the original value. Instead, we recommend the that you use the ["G17"](#GFormatString) format specifier to successfully round-trip <xref:System.Double> values.
```

Comment:
>the "R" format specifier in some cases fails to successfully round-trip the original value [](start = 35, length = 90)

This is a good comment but it may tell "R" is bad :-) can we a add a phrase telling like "R" try to optimize by formatting the number with 15 digit precisions which can fail the round tripping in some cases.

Reply:
@tarekgh, I'm not sure such a comment is useful. "R" technically is worse than "G17" in all cases. It is always less performant (due to converting back and checking if equal) and (in the cases of the known bugs) can be less accurate or completely inaccurate as well.

Reply:
Can we say, R try to format the number with 15 digits which in some cases cannot guarantee the round tripping?

Reply:
It's worth a separate discussion elsewhere, but I think if we are not going to be fixing "R" at this time, or in the future (due to back-compat) we should just deprecate it and tell people not to use it anymore because it is "bad" otherwise.

Users consuming "R" (outside of `BigInteger` where we don't think there are any bugs), are potentially introducing subtle bugs into their code and are taking a measurable performance penalty.

No matter how we word it, the issues will still exist unless we fix the underlying implementation.

Reply:
@tarekgh, @tannergooding, I left the original wording and noted the performance issue ("in some cases fails to successfully round-trip the original value and also offers relatively poor performance.")

Reply:
I am fine with that if @tannergooding is fine too.

Reply:
That sounds reasonable to me 👍